tools/autograd/derivatives.yaml

```diff
@@ -657,9 +660,7 @@
   input: zeros_like(grad)
 
 - name: prelu_backward(Tensor grad_output, Tensor input, Tensor weight, std::array<bool, 2> output_mask)
-  grad_output: zeros_like(grad_output)
-  input: zeros_like(input)
-  weight: zeros_like(weight)
+  grad_output, input, weight: prelu_double_backward(grads[0], grads[1], grad_output, input, weight, grad_mask)
```

Comment:
What?! Please tell me that it's the only case where we have this dummy zero grad... This will silently give people incorrect results! @colesbury? Why are they even marked as implemented if they're broken?

Reply:
There are a lot of other instances of `zeros_like`. They are not wired up to be user visible though. (gradcheck should catch it if we accidentally enable a zeroed grad)